openwisp_notifications/api/serializers.py

```diff
@@ -59,6 +59,7 @@ class Meta(NotificationSerializer.Meta):
         fields = [
             'id',
             'message',
+            'rendered_description',
```

Comment:
Here I am not convinced.

The API doesn't return any `description` property because we didn't use it until today, so I think it would be cleaner to define a new field called `description` which calls the `rendered_decscription` on the model instance (consult the Django REST Framework docs to see how to do this).

We'll only use the rendered description, it's highly unlikely we'll ever have to return the raw description field in the API, so let's simplify and hide this implementation details in the name of the field.

Reply:
Fixed it too